test-browser/helpers/init.js

```diff
@@ -24,7 +24,7 @@ module.exports = function (browser, callback, url, preloadPlugins = true) {
 }
 
 function initModules (browser, callback) {
-  browser.pause(5000)
+  browser.pause(20000)
```

Comment:
wow this is quite long.

Reply:
I was using it for debugging locally, i'll take it out.